js/interface.js

```diff
@@ -2610,11 +2696,47 @@ var DynamicLists = (function() {
           item.value = item.fieldValue;
         }
 
+        item.filterModifier = {
```

Comment:
Why is `filterModifier` set for all filter options when the modifier isn't necessarily always needed?